src/app/components/team/Statuses/DeleteStatusDialog.js

```diff
@@ -51,9 +51,9 @@ const DeleteStatusDialog = ({
       body={
         <React.Fragment>
           <p>
-            <FormattedMessage
+            <FormattedHTMLMessage
```

Comment:
I think you can just use `<FormattedMessage>` (after making Caio's change below). Could you please confirm?